web/src/components/contactForm.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ContactWrapper = styled.div`
+    ${tw`text-center`}
+`;
+
+const StyledForm = styled.form`
+    ${tw`flex flex-col items-center`}
+`;
+
+const StyledHeading = styled.h1`
+    ${tw`font-bold mb-8 relative text-4xl text-black-3 leading-tight`}
+`;
+const InputWrapper = styled.div`
+    ${tw`flex gap-1`}
+`;
+
+const StyledButton = styled.button`
+    ${tw`cursor-pointer bg-green py-2 px-6 text-white max-w-max mt-16 text-xl tracking-widest`}
+
+    transition: background 500ms ease-out;
+
+    // &:hover {
+    //     background: #027874;
+    // }
+`;
+
+const StyledInput = styled.input`
+    ${tw`border-2`}
+`;
+
+const StyledLabel = styled.label`
+    ${tw`flex flex-col mb-4 mx-4 text-sm`}
+`;
+
+export default function ContactForm() {
+    const [formData, setFormData] = useState(null);
+    const [submitForm, setSubmitForm] = useState(false);
```

Comment:
Can we rename this to a more accurate description of what it it? `isFormSubmitSuccessful` or something similar.